src/items/class-features/dark_matter_(su).json

```diff
@@ -39,7 +39,44 @@
       "unidentified": "",
       "value": "<p>&gt;@UUID[Compendium.sfrpg.class-features.G7p2YIGFTAU6uGjL]{Stellar Revelation}</p>\n<p><sub><img style=\"border: 1px solid var(--color-border-dark);\" src=\"systems/sfrpg/images/cup/gameplay/graviton.webp\" alt=\"\" width=\"20\" height=\"20\">&nbsp;</sub>Graviton Revelation<span></span></p>\n<p>As a move action, you can draw on the properties of dark matter to increase your density, allowing you to resist physical damage. You gain damage reduction 1/—. This increases to DR 2/— at 6th level and increases by 1 again every 3 solarian levels thereafter. This benefit lasts for 1 round or until you leave graviton mode. When you are attuned or fully attuned, your DR from dark matter is equal to half your solarian level.</p>\n<p><em>Currently, the modifier in Modifiers tab will need be to be opened then saved to update Damage Reduction. This required whenever activating/deactivating this feature, or attuning/unattuning.</em></p>"
```

Comment:
Remove the note from the end. That is no longer true.

Reply:
Didn't touch that, but I can fix that with this update.

Reply:
I know yeah, just asking you to axe it while we're working on it